Resources/doc/installation.md

```diff
@@ -0,0 +1,82 @@
+# Installation
+
+You can install this bundle with `composer` using the `sulu/validation-bundle` package.
+
+```bash
+composer require sulu/validation-bundle
+```
+
+## 1. Add SuluValidation to your project:
+
+In `composer.json`:
+
+```
+"sulu/validation-bundle": "~0.1"
```

Comment:
2 lines above the require will automatically use the latest version. i will remove this lines here.